mofo-shareprogress-page-wrapper.css

```diff
@@ -0,0 +1,270 @@
+/* All the !important rules are
+   To override CSS rules injected by Shareprogress' script */
+
+body {
+  background: #b7b9fa; /* Foudation Brand $light-blue */
+  font-family: 'Nunito Sans', sans-serif !important;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: 'Zilla Slab', serif !important;
+  font-weight: 500 !important;
+}
+
+h6,
+p {
+  font-family: 'Nunito Sans', sans-serif !important;
+}
+
+.sprog-content #share_image .step1 p {
+  font-size: 16px;
+}
+
+#header {
+  border: none;
+  background: none;
+  height: auto;
+  margin: 12px 0;
+}
+
+#header_content {
+  width: 350px !important;
+}
+
+#footer_content {
+  margin-bottom: 30px !important;
+}
+
+#footer_content .link-wrapper {
+  text-align: center;
+  margin-top: 20px;
+}
+
+#footer_content a {
+  font-family: 'Nunito Sans', sans-serif !important;
+  font-size: 12px !important;
+  color: #595cf3 !important; /* Foudation Brand $blue */
+  text-decoration: none !important;
+}
+
+#footer_content a:hover {
+  text-decoration: underline !important;
+}
+
+.sprogc-content {
+  left: 0;
+  border: none;
+  box-shadow: none;
+  padding: 0;
+}
+
+.cta {
+  font-size: 26px;
+  font-weight: 200;
+  line-height: 35.75px;
+  margin-bottom: 10px;
+  margin-top: 20px;
+  text-align: center;
+  margin: 0 auto;
+  font-family: "ZillaSlab-Light", "Arvo","Fira Sans", "Helvetica Neue", "Arial", sans-serif;
```

Comment:
we don't load any of those other fonts, probably better to cut out Arvo and Fira Sans? Or add their webfont `<link>`

Reply:
Ah good call. This is actually from the existing code. This entire chunk might not be relevant anymore. Will double check.